packages/headless-ui/vaul/test/src/app/globals.css

```
@@ -0,0 +1,18 @@
+main {
+  min-height: 500vh;
+}
+
+html {
+  height: -webkit-fill-available;
+}
+
+a {
+  text-decoration-thickness: 1px;
+  text-underline-offset: 2px;
+  text-decoration: underline;
```

Biome (2.1.2)

[error] Unexpected shorthand property text-decoration after text-decoration-thickness



(lint/suspicious/noShorthandPropertyOverrides)